LICENSES/CC-BY-SA-4.0.txt

```diff
@@ -0,0 +1,427 @@
+Attribution-ShareAlike 4.0 International
+
+=======================================================================
+
+Creative Commons Corporation ("Creative Commons") is not a law firm and
```

Comment:
Should the preamble here really be included in a license file for "Creative Commons Attribution-ShareAlike"? It seems to me that the title and the real license begins further down below?

Reply:
You might be right. 

I just followed the instructions here: https://choosealicense.com/licenses/cc-by-sa-4.0/
"How to apply this license", I guess I thought the entire text was the license. Maybe it should go.